extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -663,27 +691,57 @@ export class AmpLightboxViewer extends AMP.BaseElement {
   }
 
   /**
-   * Resizes the ImageViewer of the current slide so that the image
-   * is centered in the page. Used in open or on slide change. Also
-   * registers a onResize handler to resize the ImageViewer whenever
-   * the screen size changes.
+   * This function resizes the image inside the lightbox.
    * @return {!Promise}
    * @private
    */
-  resizeImageViewerDimensions_() {
+  onResize_() {
```

Comment:
I think the naming here is a little confusing, since this function is not always called from the `viewport#onResize` listener. I'd say an `onX` method should only be used in contexts where it is responding to an event `X`, but that is not the case on [line 610](https://github.com/ampproject/amphtml/pull/12424/files#diff-5d19ed52db55dbca977545bc8f557f36R610), [line 276](https://github.com/ampproject/amphtml/pull/12424/files#diff-5d19ed52db55dbca977545bc8f557f36R276), or [line 904](https://github.com/ampproject/amphtml/pull/12424/files#diff-5d19ed52db55dbca977545bc8f557f36R904).